modules/serverless-interacting-apps-HTTPS.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * serverless/interacting-serverless-apps.adoc
+
+[id="serverless-interacting-apps-HTTPS_{context}"]
+= Interacting with a serverless application using HTTPS
+
+Once you have deployed a serverless application, you can interact with it using HTTPS (secured route access).
+To use HTTPS, append `https://` to your application's URL as shown in the example.
+
+----
+$ curl -k https://hello-example.demo.apps.openshiftdemos.com
```

Comment:
Do we have any guidelines on which domain to use in the examples? Wouldn't example.com be better?

Reply:
Either is fine. It's up to the docs team's rule. I found several patterns:

- [apps.openshiftdemos.com](https://docs.openshift.com/container-platform/4.2/networking/ingress-operator.html#nw-installation-ingress-config-asset_configuring-ingress)
- [apps-crc.testing](https://docs.openshift.com/container-platform/4.2/serverless/knative-client.html)

and choose `apps.openshiftdemos.com`.

Reply:
OK, thanks for answer.